src/index.js

```diff
@@ -30,14 +31,23 @@ class ListOfTasks {
             <li class="w-100 p-3 border-bottom">
                 <div class="w-100 d-flex justify-content-between">
                     <div>
-                        <input type="checkbox" id="check-${task.index}">
-                        <span class="px-2">${task.description}</span>
+                        <input type="checkbox" id="check-${task.index}" ${task.completed ? 'checked' : ''}>
+                        <span id="span-${task.index}" class="px-2 ${task.completed ? 'task-completed' : ''}">${task.description}</span>
                     </div>
                     <i class="fas fa-ellipsis-v text-secondary"></i>
                 </div>
             </li>
             `;
     }
+    for (const task of this.tasks) {
```

Comment:
- [ ] OPTIONAL - Consider adding drag-and-drop sorting functionality. It would make for a good learning experience and a nice piece in your portfolio.